hydroflow/tests/compile-fail/surface_group_by_badtype_option.stderr

```diff
@@ -2,7 +2,7 @@ error[E0271]: expected `std::collections::vec_deque::IntoIter<std::option::Optio
  --> tests/compile-fail/surface_group_by_badtype_option.rs:5:9
   |
 5 |         source_iter([ Some(5), None, Some(12) ])
-  |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected tuple, found enum `Option`
+  |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected tuple, found enum `std::option::Option`
```

Comment:
this stderr test is marked as ignored so this isnt tested. left over or not rebased?

Reply:
probably not rebased I'm crap at git

Reply:
No problem either way since the test is disabled